ui/src/data_explorer/containers/DataExplorer.js

```diff
@@ -125,24 +139,32 @@ const DataExplorer = React.createClass({
 function mapStateToProps(state) {
   const {
     app: {persisted: {autoRefresh}},
-    timeRange,
-    queryConfigs,
     dataExplorer,
+    queryConfigs,
+    timeRange,
   } = state
   const queryConfigValues = _.values(queryConfigs)
 
   return {
     autoRefresh,
-    timeRange,
-    queryConfigs: queryConfigValues,
     dataExplorer,
+    queryConfigs: queryConfigValues,
+    timeRange,
   }
 }
 
 function mapDispatchToProps(dispatch) {
   return {
     handleChooseAutoRefresh: bindActionCreators(setAutoRefresh, dispatch),
     setTimeRange: bindActionCreators(viewActions.setTimeRange, dispatch),
+    showWriteFormAction: bindActionCreators(
```

Comment:
In this case, and for `hideWriteFormAction` below, since `showWriteForm` would not be competing for namespace within `<DataExplorer>` (as far as I can tell), I would just import them as `showWriteForm` and `hideWriteForm`